release/ray_release/scripts/ray_bisect.py

```diff
@@ -27,11 +27,21 @@
         "capacity, but reduce the bisect duration"
     ),
 )
+@click.option(
+    "--flaky-rerun",
+    default=1,
+    type=int,
+    help=(
+        "The number of time we run test on the same commit, to account for test "
+        "flakiness. Commit passes only when it passes on all runs"
+    ),
+)
 def main(
     test_name: str,
     passing_commit: str,
     failing_commit: str,
-    concurrency: Optional[int] = 1,
+    concurrency: int = 1,
+    flaky_rerun: int = 1,
```

Comment:
maybe change this to `retry_on_fail: int`? or maybe just `flaky: bool` and when it is flaky, retry with max of 3 times?

Reply:
it's not really retrying - it will always run the commit exactly that many time, and mark the commit as fail even with just one failure

Reply:
then why is it called `flaky_rerun`? if it is 1, is it still flaky?

Reply:
It's a bad name ;)), maybe just rerun_per_commit or something - totally, I'll poke you in the office to explain more